include/sys/time.h

```diff
@@ -51,6 +59,21 @@ typedef struct bintime {
     .sec = 0, .frac = ((1ULL << 63) / (hz)) << 1                               \
   }
 
+static inline time_t tm2sec(tm_t t) {
+  time_t res = 0;
+  int month_int_days[13] = {0,   0,   31,  59,  90,  120, 151,
```

Comment:
Why not `static const`?

Reply:
My bad. Thank you!

Reply:
@pj1031999 `const` OK, but why `static` here?